### PR TITLE
Introduce --skip-checks in openqa-clone-job

### PR DESCRIPTION
**Pull request by grisu48**
Introduces the `--skip-checks` parameter to avoid additional checks like maintenance update availability in `openqa-clone-job`.

### CODE REVIEW

File: script/openqa-clone-job [hunk 1]
```diff
@@ -149,7 +153,6 @@ https://openqa.opensuse.org 123456
 
 Will create 50 clones of the same job.
 
-
```
Comment (okurz) [inline script/openqa-clone-job:152]: Possibly unrelated change. I suggest you just keep it as is or create a separate commit (not separate PR) stating what this fixes. Did you see a problem in the generated help text?

File: script/openqa-clone-job [hunk 2]
```diff
@@ -104,6 +104,10 @@ from JOBREF if it is a URL).
 Specifies the directory to store test assets (defaults to
 $OPENQA_SHAREDIR/factory).
 
+=item B<--skip-checks>
+
+Skips additional checks.
```
Comment (okurz) [inline script/openqa-clone-job:109]: I think you can explicitly state, e.g.

```suggestion
Skips additional checks like maintenance update availability.
```

**Review by okurz — CHANGES_REQUESTED** [review]

File: script/openqa-clone-job (hunk 1 above)
Comment (grisu48) [reply script/openqa-clone-job:152]: Will add it as separate commit though.

File: script/openqa-clone-job (hunk 2 above)
Comment (grisu48) [reply script/openqa-clone-job]: done.

File: script/openqa-clone-job [hunk 3]
```diff
@@ -104,6 +104,10 @@ from JOBREF if it is a URL).
 Specifies the directory to store test assets (defaults to
 $OPENQA_SHAREDIR/factory).
 
+=item B<--skip-checks>
+
+Skips additional checks like maintenance update availability.
```
Comment (kalikiana) [inline script/openqa-clone-job:109]: **Introduce --skip-checks** where? The PR title is ambiguous.

I can see a change to `openqa-clone-job`?

Comment (grisu48) [reply script/openqa-clone-job:109]: I added "in openqa-clone-job" in both the PR title and the corresponding commit message.

**Comment by grisu48** [comment]
I tidied the code.

**Review by okurz — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Review by kalikiana — APPROVED** [review]
Nice addition!
